chat_exporter/ext/html_generator.py

```diff
@@ -35,7 +35,7 @@ async def fill_out(guild, base, replacements):
         elif mode == PARSE_MODE_EMOJI:
             v = await ParseMarkdown(v).special_emoji_flow()
 
-        base = base.replace("{{" + k + "}}", v)
+        base = base.replace("{{" + k + "}}", v.strip())
```

Comment:
Idk if this is really needed, but it shouldn't hurt I guess. Was there a specific reason why you added it?

Reply:
Yes its needed because for some reason ids, such as user ids in `data` attributes, would have whitespace, which breaks the javascript for things like User Profiles etc as when it was looking up the class via the data attribute it wouldnt find it (because of the extra white space)

Before:
![image](https://github.com/mahtoid/DiscordChatExporterPy/assets/70772334/b66617f1-74be-495d-a15b-bf337c9183f5)

After:
![image](https://github.com/mahtoid/DiscordChatExporterPy/assets/70772334/b8dae257-9572-48c6-87d8-50e59fd31f61)